hpedockerplugin/hpe/hpe3par_opts.py

```diff
@@ -48,6 +49,13 @@
                      "standard dict config form: replication_device = "
                      "target_device_id:<required>,"
                      "key1:value1,key2:value2..."),
+    cfg.StrOpt('hpe3par_default_fpg_size',
+               default='32T',
```

Comment:
@imran-ansari We need to reconsider the default size, i think size 32TiB is for 3.2.2  3PAR OS whereas for 3.3.1 it is 64TiB

Reply:
Earlier I was using FTC array which has 3.2.2 3PAR OS and that's the reason 32T was used. Now that we have 3.3.1 3PAR OS in Bangalore, will change it to 64T